src/lib/replication/service/repl_service_impl.cpp

```diff
@@ -19,19 +19,26 @@
 #include "replication/repl_dev/solo_repl_dev.h"
 
 namespace homestore {
-ReplicationServiceImpl& repl_service() { return hs()->repl_service(); }
+ReplicationService& repl_service() { return hs()->repl_service(); }
 
-ReplicationServiceImpl::ReplicationServiceImpl(repl_impl_type impl_type, std::unique_ptr< ReplServiceCallbacks > cbs) :
-        m_svc_cbs{std::move(cbs)}, m_repl_type{impl_type} {
+ReplicationServiceImpl::ReplicationServiceImpl(repl_impl_type impl_type) : m_repl_type{impl_type} {
     meta_service().register_handler(
-        "replication",
+        "repl_dev",
```

Comment:
Should we add group_id along with "repl_dev" for uniqueness of this metablk client?